src/lib/utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Make a link external.
+ *
+ * Usage:
+ * ```
+ * <a href="https://example.com/" {...external}>Example</a>
+ * ```
+ */
+export const external: Partial<HTMLAnchorElement> = {
```

Comment:
Please rename this to something more unique and identifiable, external on it's does not tell me much what it does.

Reply:
Considering its usage and the JSDoc, I think it's pretty clear what it does.

Reply:
I have to disagree with that, importing `external` from `$lib` is not clear on what it does and if I have to look at other usage of it to know what it does, that is also not a good pattern in my opinion.

Reply:
is `link` a better name?

Reply:
I would use `externalLinkProperties` since that is exactly what it contains.

Reply:
Maybe just `externalLink`? That name is a bit long.